spec/system/task_spec.rb

```diff
@@ -1,24 +1,28 @@
 require 'rails_helper'
 
 RSpec.describe 'Task', type: :system do
+  let(:project) { create(:project) }
+  let(:task) { create(:task) }
+  let(:another_task) { create(:task, :another_task) }
+
   describe 'Task一覧' do
     context '正常系' do
       it '一覧ページにアクセスした場合、Taskが表示されること' do
         # TODO: ローカル変数ではなく let を使用してください
-        project = FactoryBot.create(:project)
-        task = FactoryBot.create(:task, project_id: project.id)
+        task
         visit project_tasks_path(project)
         expect(page).to have_content task.title
         expect(Task.count).to eq 1
         expect(current_path).to eq project_tasks_path(project)
       end
 
-      xit 'Project詳細からTask一覧ページにアクセスした場合、Taskが表示されること' do
+      it 'Project詳細からTask一覧ページにアクセスした場合、Taskが表示されること' do
         # FIXME: テストが失敗するので修正してください
-        project = FactoryBot.create(:project)
-        task = FactoryBot.create(:task, project_id: project.id)
+        task
         visit project_path(project)
         click_link 'View Todos'
+        windows = page.driver.browser.window_handles
+        page.driver.browser.switch_to.window(windows.last)
```

Comment:
細かいので修正不要ですが、1行でも記載できますね！
```suggestion
        switch_to_window(windows.last)
```

Reply:
48b229d にて修正しました。